crowdsourcing/wpconvlib/src/conversation.test.ts

```diff
@@ -148,32 +153,68 @@ export class ConversationTest {
       .beFalse();
   }
 
+  public "Map of structureConversaton with deleted comments"() {
+    const rootComment =
+      conversation.structureConversaton(
+        JSON.parse(JSON.stringify(example_conversation5)));
```

Comment:
What's the JSON.parse(JSON.stringify()) for? Are you trying to copy the object?

Reply:
@iislucas I copied this one as from other functions. Maybe you know better about this?

Reply:
This is a fairly standard way to copy a JSON object. I imagine underscore library has some nicer ways to do this, but which are probably implement using JSON... do you have a recommendation for a different way to copy objects Rachel?

Reply:
This is fine, just wanted to confirm that was the intention here!